cpp/tests/t/pipelines/slac/SLAC.cpp

```diff
@@ -86,8 +85,7 @@ TEST_P(SLACPermuteDevices, DISABLED_RunSLACOptimizerForFragments) {
         utility::LogError(
                 "No fragment found in {}, please make sure the test dataset "
                 "has been downloaded in "
-                "open3d_downloads/tests/reconstruction_system/ been "
-                "downloaded.",
+                "test_data/open3d_downloads/tests/reconstruction_system/.",
```

Comment:
we may use `GetDataPathDownload` instead of `test_data/open3d_downloads`.

Reply:
Done.